### PR TITLE
Support target versions for DB upgrades

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: infrastructure/tooling/src/main.js [hunk 1]
```diff
@@ -147,6 +147,7 @@ program.command('dev:init')
 
 program.command('dev:db:upgrade')
   .description('Run `yarn db:upgrade` for a development environment')
+  .option('--db-version <v>', 'Downgrade to this DB version (optional, defaults to latest)')
```
Comment (helfi92) [inline infrastructure/tooling/src/main.js:150]: Wouldn't this defeat the purpose of having a dev:db:downgrade?

**Comment by helfi92** [comment]
Tests seem to be failing.

File: infrastructure/tooling/src/main.js (hunk 1 above)
Comment (djmitche) [reply infrastructure/tooling/src/main.js:150]: Whoops, that was copy-paste -- should be 
```suggestion
  .option('--db-version <v>', 'Upgrade to this DB version (optional, defaults to latest)')
```

**Comment by djmitche** [comment]
Should be fixed.. let's see what the tests say

**Pull request merged**